### DISCUSSION TITLE
Examples of LWP3

### DISCUSSION THREAD

**Discussion by novafox3c** [pybricks/support#914] [General]
I have a Lego Move and City Hub and I was wondering how to turn the Move Hub into a remote control for the City Hub. Do you have any examples using the LWP3 Protocol?

**Comment by dlech** [comment]
I'm not aware of any examples. The `pybricksdev` command line tool has a `lwp3 repl` command that has code completion. It useful for exploring the LWP3 protocol.

**Comment by novafox3c** [comment]
I have one question. Can the Robot Inventor Hub send messages via the LWP3 Protocol?

**Reply by dlech** [reply]
Using the Pybricks firmware, yes, it should be able to.
